### PR TITLE
Changed callbacks to Async await

### LINKED ISSUE #3: Use async/await for processing requests
Right now we're using promises and/or callbacks. Read this: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await

An example:
```Javascript
app.get(`/${apiPrefix}/todos`, async (req, res) => {
  const q = 'SELECT * FROM todos';

  try {
    const result = await pool.query(q);
    res.status(200).json(result.rows);
  } catch (err) {
    throw err;
  }
});
```

### PR DESCRIPTION
**Pull request by opmorgan**
resolve #3

### CODE REVIEW

File: server/routes/todo.js [hunk 1]
```diff
@@ -55,37 +63,40 @@ export default function (router) {
...
-                } else {
-                    console.log("Sucessfully deleted todo!", result);
-                }
-            })
+
+            const deleteCommand = "DELETE FROM todos WHERE id = $1;";
+
+            try {
+                const result = await pool.query(deleteCommand, [id]);
+                console.log("Sucessfully deleted todo!", result);
```
Comment (ttylta) [inline server/routes/todo.js]: You need to send a response here. Something akin to `res.status(200)...` like you did above.

File: server/routes/todo.js [hunk 2]
```diff
@@ -9,41 +9,49 @@ export default function (router) {
      * Returns all of the todos from the database.
      */
     router.get("/todos", async (req, res) => {
-        pool.query('SELECT * FROM todos', (err, result) => {
-            if (err) {
-                throw err;
-            }
 
+        const selectAllCommand = "SELECT * FROM todos";
```
Comment (ttylta) [inline server/routes/todo.js]: This is a nitpick, but technically it's not a command. It's a query. I would amybe changed the variable to `selectAllQuery` or more simply: `query`.

File: server/routes/todo.js (hunk 1 above)
Comment (balbyu) [reply server/routes/todo.js]: All routes now send appropriate responses

File: server/routes/todo.js (hunk 2 above)
Comment (balbyu) [reply server/routes/todo.js]: Renamed all complicated variable commands to simpler "query"

**Pull request merged**